tools/ci_build/github/azure-pipelines/py-packaging-pipeline.yml

```diff
@@ -55,6 +55,12 @@ parameters:
    - RelWithDebInfo
    - MinSizeRel
 
+# Only applies to QNN packages.
+- name: qnn_sdk_version
```

Comment:
@snnn I added the ability to specify qnn sdk version to the python packaging pipeline. I just want to check with you that this is ok.

Reply:
If you need to support multiple SDK versions, I think it would be better to split the QNN jobs out. They are independent. Then instead of making the version number as a configurable parameter, create multiple pipelines with different settings, and run each of them every night to get the best coverage. We handle CUDA EP in this way.

Reply:
it's a slightly different requirement than CUDA. we won't publish multiple packages of the same ORT version against different QNN versions.
It's primarily so we have flexibility to choose which version to publish against (and not have to update any yml files etc.)
we are also going to start distributing QNN SDK along with the ORT-QNN packages as Qualcomm has provided us the redist permissions.

Reply:
@snnn @jywu-msft Would we be able to merge this now?

Reply:
let's merge this for now.